src/forge/controller/launcher.py

```diff
@@ -206,7 +206,7 @@ def __init__(
         self.timeout_sec = 1 * 60 * 60  # Kill the job if idle for 1 hour
         self.user = getpass.getuser()
         self.work_dir = f"/home/{self.user}"
```

Comment:
This is hardcoded and could you check if it works when launching from `fbsource/fbcode/pytorch/torchforge`? I need to do the following https://www.internalfb.com/diff/D88381552 (see the launcher.py)

Reply:
I don't think it will work from `fbsource/fbcode/pytorch/torchforge` because the path is different, but looks like your diff should fix it